src/CattyTests/DataModel/UserDataContainerTest.swift

```diff
@@ -430,19 +431,19 @@ final class UserDataContainerTest: XCTestCase {
         let list3 = UserList(name: "testName3")
         let list4 = UserList(name: "testName4")
 
-        var result = container.addObjectList(list1, for: objectA)
+        var result = objectA.userData.addList(list1)
         XCTAssertTrue(result)
 
-        result = container.addObjectList(list2, for: objectB)
+        result = objectB.userData.addList(list2)
         XCTAssertTrue(result)
 
-        container.programListOfLists.add(list3 as Any)
+        container.addList(list3)
 
-        XCTAssertTrue(container.getUserListNamed(list1.name, for: objectA)?.isEqual(list1) == true)
-        XCTAssertTrue(container.getUserListNamed(list1.name, for: objectB) == nil)
-        XCTAssertTrue(container.getUserListNamed(list3.name, for: objectA)?.isEqual(list3) == true)
-        XCTAssertTrue(container.getUserListNamed(list3.name, for: objectB)?.isEqual(list3) == true)
-        XCTAssertTrue(container.getUserListNamed(list4.name, for: objectA) == nil)
+        XCTAssertTrue(objectA.userData.getUserList(withName: list1.name)?.isEqual(list1) == true)
+        XCTAssertTrue(objectB.userData.getUserList(withName: list1.name) == nil)
+        XCTAssertTrue(container.getUserList(withName: list3.name)?.isEqual(list3) == true)
+        XCTAssertTrue(container.getUserList(withName: list3.name)?.isEqual(list3) == true)
```

Comment:
This line is duplicated but there is an assert missing for `objectB` and `list2`.